accessibility/accessibility-in-pdfs.md

```diff
@@ -0,0 +1,94 @@
+# Accessibility in PDFs
+
+## A warning
+
+We specialise in web accessibility, not PDF accessibility. We can give you some pointers, but following this advice is **not** a guarantee that your document will be accessible. 
```

Comment:
```suggestion
```
Implicit. Also, given the context of the FED Playbook, who "we" are is indistinct.